lockbox-ios/View/ItemListView.swift

```diff
@@ -76,17 +85,44 @@ extension ItemListView: ItemListViewProtocol {
 
 extension ItemListView {
     fileprivate func setupDataSource() {
-        self.dataSource = RxTableViewSectionedReloadDataSource<ItemSectionModel>(
-                configureCell: { _, tableView, _, item in
-                    guard let cell = tableView.dequeueReusableCell(withIdentifier: "itemlistcell") as? ItemListCell else { // swiftlint:disable:this line_length
-                        fatalError("couldn't find the right cell!")
+        self.dataSource = RxTableViewSectionedAnimatedDataSource<ItemSectionModel>(
+                configureCell: { dataSource, tableView, path, _ in
+                    let cellConfiguration = dataSource[path]
+
+                    var retCell: UITableViewCell
+                    switch cellConfiguration {
+                    case .Search:
+                        guard let cell = tableView.dequeueReusableCell(withIdentifier: "filtercell") as? FilterCell,
```

Comment:
Are these cells `register`ed with the tableView anywhere?

Reply:
storyboard.